control/frdata.py

```diff
@@ -342,110 +336,107 @@ def __pow__(self, other):
+    def __call__(self, s, squeeze=True):
+        """Evaluate system's transfer function at complex frequencies.
+        
+        Returns the complex frequency response `sys(s)`.
 
-        Reports the value of the magnitude, phase, and angular frequency of
-        the requency response evaluated at omega, where omega is a list of
-        angular frequencies, and is a sorted version of the input omega.
+        To evaluate at a frequency omega in radians per second, enter 
+        x = omega*1j or use FRD.eval(omega)
 
         Parameters
         ----------
-        omega : array_like
-            A list of frequencies in radians/sec at which the system should be
-            evaluated. The list can be either a python list or a numpy array
-            and will be sorted before evaluation.
-
+        s: complex scalar or array_like 
+            Complex frequencies 
+        squeeze: bool, optional (default=True)
+            If True and sys is single input single output (SISO), returns a 
+            1D array or scalar depending on the length of x. 
+            
         Returns
         -------
-        mag : (self.outputs, self.inputs, len(omega)) ndarray
-            The magnitude (absolute value, not dB or log10) of the system
-            frequency response.
-        phase : (self.outputs, self.inputs, len(omega)) ndarray
-            The wrapped phase in radians of the system frequency response.
-        omega : ndarray or list or tuple
-            The list of sorted frequencies at which the response was
-            evaluated.
-        """
-        # Preallocate outputs.
-        numfreq = len(omega)
-        mag = empty((self.outputs, self.inputs, numfreq))
-        phase = empty((self.outputs, self.inputs, numfreq))
+        fresp : (num_outputs, num_inputs, len(x)) or len(x) complex ndarray
+            The frequency response of the system. Array is len(x) if and only if
+            system is SISO and squeeze=True.
 
-        omega.sort()
-
-        for k, w in enumerate(omega):
-            fresp = self._evalfr(w)
-            mag[:, :, k] = abs(fresp)
-            phase[:, :, k] = angle(fresp)
+        Raises
+        ------
+        ValueError
+            If `s` is not purely imaginary, because FrequencyDomainData systems
+            are only defined at imaginary frequency values. 
 
-        return mag, phase, omega
+        """
+        if any(abs(np.array(s, ndmin=1).real) > 0):
+            raise ValueError("__call__: FRD systems can only accept"
+                            "purely imaginary frequencies")
+        # need to preserve array or scalar status
+        if hasattr(s, '__len__'):
+            return self.eval(np.asarray(s).imag, squeeze=squeeze)
+        else:
+            return self.eval(complex(s).imag, squeeze=squeeze)
+        
+    def freqresp(self, omega):
+        warn("FrequencyResponseData.freqresp(omega) will be deprecated in a "
```

Comment:
This means, that the method is displayed with an empty description in the docs.

![Screenshot_20200818_121921](https://user-images.githubusercontent.com/4623504/90501809-3409a480-e14d-11ea-9bfd-49dd27dc308e.png)

We could either copy the docstring from `frequency_response` with
```
FrequencyResponseData.freqresp.__doc__ = FrequencyResponseData.frequency_response.__doc__
```
or just the summary line and a note.

Adding a [`..deprecated::`](https://numpydoc.readthedocs.io/en/latest/format.html#sections) Sphinx directive could also be useful, but that is for real deprecations only, not pending ones. (Why is this pending anyway? We are effectively deprecating it already)

Reply:
Done in new commit. Re deprecation: makes sense I'll mark it as deprecated instead of pending deprecation.